hugegraph-core/src/main/java/com/baidu/hugegraph/backend/serializer/BinarySerializer.java

```diff
@@ -600,13 +621,12 @@ private Query writeQueryEdgeRangeCondition(ConditionQuery cq) {
         }
         Id label = cq.condition(HugeKeys.LABEL);
 
-        int size = 1 + vertex.length() + 1 + label.length() + 16;
-        BytesBuffer start = BytesBuffer.allocate(size);
-        start.writeId(vertex);
+        BytesBuffer start = BytesBuffer.allocate(BytesBuffer.BUF_EDGE_ID);
+        writePartitionedId(HugeType.EDGE, vertex, start);
```

Comment:
for om.baidu.hugegraph.core.EdgeCoreTest test error

writeQueryEdgeRangeCondition() add writePartitionedId()